exporter/coralogixexporter/testdata/config.yaml

```diff
@@ -31,3 +30,14 @@ coralogix/all:
   application_name: "APP_NAME"
   subsystem_name: "SUBSYSTEM_NAME"
   timeout: 5s
+
+coralogix/domain:
```

Comment:
Since validation currently allows both domain and signal-specific endpoints at the same time, can you add a test scenario for that?

If configuring both at the same time isn't allowed can you add a validation for that?

Reply:
@povilasv awesome, can you also add a new test case in `testdata/config.yaml`/`config_test.go` that covers setting both fields?  This test would ensure that the scenario passes config validation.